src/app/workspace/resource/services/resource.service.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { TestBed } from '@angular/core/testing';
+import { AppInitService } from 'src/app/app-init.service';
+import { ResourceService } from './resource.service';
+
+describe('ResourceService', () => {
+    let service: ResourceService;
+
+    const appInitSpy = {
+        dspAppConfig: {
+            iriBase: 'http://rdfh.ch'
+        }
+    };
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            providers: [
+                {
+                    provide: AppInitService,
+                    useValue: appInitSpy
+                }
+            ]
+        });
+        service = TestBed.inject(ResourceService);
+    });
+
+    it('should be created', () => {
+        expect(service).toBeTruthy();
+    });
```

Comment:
I would add some tests here for the service methods. They can be simple tests but since this service is quite crucial to the whole app running, it would be good to ensure these methods are working as intended.

Reply:
Done in f4a6b170